From 7e4a8d9e4a7a55324cbd2c7e557c2d1f5e229019 Mon Sep 17 00:00:00 2001
From: Jianqun xu <jay.xu@rock-chips.com>
Date: Fri, 4 Dec 2015 09:36:09 +0800
Subject: [PATCH] dt-binding: rockchip: add IDs for RK3368 SoCs power-domains

In order to meet low power requirements, a power management unit (PMU)
is designed for controlling power resources in RK3368. This patch adds
power-domain IDs to for further use.

Change-Id: Id547f801c6a072957a73743a8397f732bd0c88d9
Signed-off-by: Jianqun xu <jay.xu@rock-chips.com>
---
 include/dt-bindings/power/rk3368-power.h | 28 ++++++++++++++++++++++++
 1 file changed, 28 insertions(+)
 create mode 100644 include/dt-bindings/power/rk3368-power.h

diff --git a/include/dt-bindings/power/rk3368-power.h b/include/dt-bindings/power/rk3368-power.h
new file mode 100644
index 000000000000..2a1222bce4ea
--- /dev/null
+++ b/include/dt-bindings/power/rk3368-power.h
@@ -0,0 +1,28 @@
+#ifndef __DT_BINDINGS_POWER_RK3368_POWER_H__
+#define __DT_BINDINGS_POWER_RK3368_POWER_H__
+
+/* VD_CORE */
+#define PD_A53_L0	0
+#define PD_A53_L1	1
+#define PD_A53_L2	2
+#define PD_A53_L3	3
+#define PD_SCU_L	4
+#define PD_A53_B0	5
+#define PD_A53_B1	6
+#define PD_A53_B2	7
+#define PD_A53_B3	8
+#define PD_SCU_B	9
+
+/* VD_LOGIC */
+#define PD_BUS		10
+#define PD_PERI		11
+#define PD_VIO		12
+#define PD_ALIVE	13
+#define PD_VIDEO	14
+#define PD_GPU_0	15
+#define PD_GPU_1	16
+
+/* VD_PMU */
+#define PD_PMU		17
+
+#endif
-- 
2.35.3

